### PR TITLE
Bmw 631/add more info

### PR DESCRIPTION
**Pull request by AlexSeongJu-sr**
I feel like 
map<string, string> extra_info 
is rather messy. 
 
so I just added sensor_name and topic.

### CODE REVIEW

**Review by thor-sr — COMMENTED** [review]
This will break backwards compatibility, no?

File: output.proto
Comment (young-seoulrobotics) [inline output.proto:68]: @thor-sr @sebastian-seoul-robotics FYI, we output pairs of ROS topic and sensor status, where users have no idea about which topic corresponds to which lidar. Alex is changing the key to use the UID and putting extra info (e.g., lidar name in SENSR and topic name for backward compatibility). Here's the link to the relevant conversation: https://seoulrobotics.atlassian.net/browse/BMW-631?focusedCommentId=32983

File: output.proto
Comment (young-seoulrobotics) [inline output.proto:70]: Could you move `status` to the top and set 1 to the id? It's subtle but I would like users to think the status is the main information in the message.

**Comment by AlexSeongJu-sr** [comment]
image
when changed with this proto msg, output will look like this.

File: output.proto
Comment (AlexSeongJu-sr) [reply output.proto]: yup

**Comment by sebastian-seoul-robotics** [comment]
> This will break backwards compatibility, no?

It will break backwards compatibility. As long as it not applied to sensr_i_release/3.1.x, there should not be a problem. The next release will be a major update (4.0). So no backwards compatibility required

@mogandaz fyi

**Comment by AlexSeongJu-sr** [comment]
so I guess it is fine to merge to master as long as I don't merge to sensr_i_release. 
could you approve ?

**Review by thor-sr — APPROVED** [review]

File: output.proto
Comment (young-seoulrobotics) [inline output.proto:73]: Just to be safe, let's reserve 3 and change the ID to 4. Reusing the id is strictly forbidden. Then this would not break the existing client applications.

**Review by sebastian-seoul-robotics — APPROVED** [review]

**Pull request merged**
